src/main/java/org/bytedeco/javacv/Frame.java

```diff
@@ -128,6 +132,7 @@ public Frame(int width, int height, int depth, int channels, int imageStride) {
         this.image = new Buffer[1];
         this.data = null;
         this.streamIndex = -1;
+        this.pictType = (char) av_get_picture_type_char(AV_PICTURE_TYPE_NONE);
```

Comment:
Let's just set that to 0 here to prevent a dependency on FFmpeg.

Reply:
`av_get_picture_type_char` also depend on FFmpeg. 
How about init pictType with `'N'` or just `''`

Reply:
`''` isn't a valid char literal, you probably mean `'\0'`, which is the same as just `0`.

Reply:
Got it! Done.